deployments/ansible/roles/collector/meta/main.yml

```diff
@@ -21,7 +21,6 @@ galaxy_info:
       versions:
         - bionic
         - focal
-        - xenial
```

Comment:
I think this is removing 16.0 as supported platform version from the actual ansible role. Do we want to do this?

Reply:
Good catch! I've reverted this change